apps/molgenis-components/src/components/forms/FormGroup.vue

```diff
@@ -72,7 +74,7 @@ export default {
       <label for="my-id2">required field form group</label>
       <FormGroup id="my-id2"
         label="my label"
-        required
+        required="'true'"
```

Comment:
Please also add an example which uses the boolean true, so we can see that both work :)